pyinstaller.py

```diff
@@ -19,7 +19,6 @@
 import platform
 import shutil
 import subprocess
-from distutils import dir_util
```

Comment:
Further down there's the following line:
```
"--hidden-import=distutils.dir_util "
```
Could that go as well?

Reply:
I think those can be removed, they are legacy from Conan 1.X. I have checked with the ci job that generates the installers and everything works removing both the imports and the "--hidden-import=distutils.dir_util" line. There we use 3.6 in Windows and 3.8 and 3.9 in Linux and Macos respectively. So I think these changes should be fine.

Reply:
I tried it over Python 3.11 and 3.12. Both are working correctly.